[collada-dom] Build error

Package: collada-dom[core]:x64-windows -> 2.5.0#9

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.29.30141.0
-    vcpkg-tool version: 2023-04-07-bedcba5172f5e4b91caac660ab7afe92c27a9895
    vcpkg-scripts version: 1e334d770 2023-06-07 (13 hours ago)

**To Reproduce**

`vcpkg install libzip:x64-windows`
`vcpkg install collada-dom:x64-windows`

**Failure logs**

```
-- Downloading https://github.com/rdiankov/collada-dom/archive/d37ae7532e350b87c88712e9f6ab4b1f440d20cd.tar.gz -> rdiankov-collada-dom-d37ae7532e350b87c88712e9f6ab4b1f440d20cd.tar.gz...
-- Extracting source D:/cschulte/DevEnv64/vcpkg/downloads/rdiankov-collada-dom-d37ae7532e350b87c88712e9f6ab4b1f440d20cd.tar.gz
-- Applying patch vs-version-detection.patch
-- Applying patch use-uriparser.patch
-- Applying patch use-vcpkg-minizip.patch
-- Applying patch fix-shared-keyword.patch
-- Applying patch fix-emscripten.patch
-- Using source at D:/cschulte/DevEnv64/vcpkg/buildtrees/collada-dom/src/1f440d20cd-258c30aa60.clean
-- Found external ninja('1.10.2').
-- Configuring x64-windows
-- Building x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: D:/cschulte/DevEnv64/vcpkg/downloads/tools/cmake-3.25.1-windows/cmake-3.25.1-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j9
    Working Directory: D:/cschulte/DevEnv64/vcpkg/buildtrees/collada-dom/x64-windows-dbg
    See logs for more information:
      D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\install-x64-windows-dbg-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/collada-dom/portfile.cmake:19 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\install-x64-windows-dbg-out.log</summary>

```
[1/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\dae\daeArray.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeArray.cpp
[2/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\src\1.5\CMakeFiles\colladadom150.dir\dom\domKinematics_connect_param.cpp.obj /Fddom\src\1.5\CMakeFiles\colladadom150.dir\colladadom150.pdb /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\1.5\dom\domKinematics_connect_param.cpp
[3/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\src\1.5\CMakeFiles\colladadom150.dir\dom\domKinematics_limits.cpp.obj /Fddom\src\1.5\CMakeFiles\colladadom150.dir\colladadom150.pdb /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\1.5\dom\domKinematics_limits.cpp
[4/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\src\1.5\CMakeFiles\colladadom150.dir\dom\domKinematics_frame.cpp.obj /Fddom\src\1.5\CMakeFiles\colladadom150.dir\colladadom150.pdb /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\1.5\dom\domKinematics_frame.cpp
...
Skipped 125 lines
...

[122/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\dae\daeMetaGroup.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeMetaGroup.cpp
[123/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\dae\daeRefCountedObj.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeRefCountedObj.cpp
[124/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\dae\daeMetaElementAttribute.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeMetaElementAttribute.cpp
[125/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\dae\daeTinyXMLPlugin.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeTinyXMLPlugin.cpp
[126/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\dae\daeStringRef.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeStringRef.cpp
[127/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\dae\daeStringTable.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeStringTable.cpp
[128/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\modules\stdErrPlugin\stdErrPlugin.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\stdErrPlugin\stdErrPlugin.cpp
[129/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\dae\daeMetaElement.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeMetaElement.cpp
[130/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\dae\daeRawResolver.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeRawResolver.cpp
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeRawResolver.cpp(45): warning C4996: 'fopen': This function or variable may be unsafe. Consider using fopen_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
[131/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\dae\daeStandardURIResolver.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeStandardURIResolver.cpp
[132/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\dae\daeSIDResolver.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeSIDResolver.cpp
[133/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\dae\daeURI.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeURI.cpp
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeURI.cpp(324): warning C4996: 'strcpy': This function or variable may be unsafe. Consider using strcpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
[134/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\dae\daeUtils.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeUtils.cpp
[135/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\dae\daeZAEUncompressHandler.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeZAEUncompressHandler.cpp
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeZAEUncompressHandler.cpp(274): warning C4996: 'boost::filesystem::path::branch_path': Use path::parent_path() instead
[136/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\dae\daeDom.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\dae\daeDom.cpp
[137/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\modules\STLDatabase\daeSTLDatabase.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\STLDatabase\daeSTLDatabase.cpp
[138/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp
FAILED: dom/CMakeFiles/collada-dom.dir/src/modules/LIBXMLPlugin/daeLIBXMLPlugin.cpp.obj 
C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -Dcollada_dom_EXPORTS -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -external:W0 /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\CMakeFiles\collada-dom.dir\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp.obj /Fddom\CMakeFiles\collada-dom.dir\ /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(21): error C2182: 'fill_win32_filefunc' : utilisation non conforme du type 'void'
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(21): error C2146: erreur de syntaxe : absence de ';' avant l'identificateur 'OF'
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(21): error C2065: 'zlib_filefunc_def' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(21): error C2065: 'pzlib_filefunc_def' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(22): error C2182: 'fill_win32_filefunc64' : utilisation non conforme du type 'void'
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(22): error C2146: erreur de syntaxe : absence de ';' avant l'identificateur 'OF'
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(22): error C2065: 'zlib_filefunc64_def' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(22): error C2065: 'pzlib_filefunc_def' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(22): error C2374: 'OF' : redéfinition ; initialisation multiple
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(21): note: voir la déclaration de 'OF'
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(23): error C2182: 'fill_win32_filefunc64A' : utilisation non conforme du type 'void'
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(23): error C2146: erreur de syntaxe : absence de ';' avant l'identificateur 'OF'
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(23): error C2065: 'zlib_filefunc64_def' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(23): error C2065: 'pzlib_filefunc_def' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(23): error C2374: 'OF' : redéfinition ; initialisation multiple
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(21): note: voir la déclaration de 'OF'
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(24): error C2182: 'fill_win32_filefunc64W' : utilisation non conforme du type 'void'
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(24): error C2146: erreur de syntaxe : absence de ';' avant l'identificateur 'OF'
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(24): error C2065: 'zlib_filefunc64_def' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(24): error C2065: 'pzlib_filefunc_def' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(24): error C2374: 'OF' : redéfinition ; initialisation multiple
D:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip\iowin32.h(21): note: voir la déclaration de 'OF'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(104): error C3646: 'zf' : spécificateur de substitution inconnu
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(104): error C4430: spécificateur de type manquant - int est pris en compte par défaut. Remarque : C++ ne prend pas en charge int par défaut
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(92): error C2065: 'zf' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(95): error C2065: 'zf' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(96): error C2065: 'zf' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(96): error C3861: 'zipClose' : identificateur introuvable
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(97): error C2065: 'ZIP_OK' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(453): error C2065: 'zlib_filefunc64_def' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(453): error C2146: erreur de syntaxe : absence de ';' avant l'identificateur 'ffunc'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(453): error C2065: 'ffunc' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(454): error C2065: 'ffunc' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(455): error C2039: 'zf' n'est pas membre de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(88): note: voir la déclaration de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(455): error C2065: 'APPEND_STATUS_CREATE' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(455): error C2065: 'ffunc' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(455): error C3861: 'zipOpen2_64' : identificateur introuvable
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(459): error C2039: 'zf' n'est pas membre de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(88): note: voir la déclaration de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(468): error C2065: 'zip_fileinfo' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(468): error C2146: erreur de syntaxe : absence de ';' avant l'identificateur 'zi'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(468): error C2065: 'zi' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(469): error C2065: 'zi' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(470): error C2065: 'zi' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(471): error C2065: 'zi' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(472): error C2065: 'zi' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(473): error C2065: 'zi' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(474): error C2065: 'zi' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(475): error C2065: 'zi' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(476): error C2065: 'zi' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(477): error C2065: 'zi' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(484): error C2039: 'zf' n'est pas membre de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(88): note: voir la déclaration de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(484): error C2065: 'zi' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(484): error C2065: 'Z_DEFLATED' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(484): error C2065: 'MAX_WBITS' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(484): error C2065: 'DEF_MEM_LEVEL' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(484): error C2065: 'Z_DEFAULT_STRATEGY' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(484): error C3861: 'zipOpenNewFileInZip3_64' : identificateur introuvable
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(485): error C2065: 'ZIP_OK' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(492): error C2039: 'zf' n'est pas membre de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(88): note: voir la déclaration de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(492): error C3861: 'zipWriteInFileInZip' : identificateur introuvable
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(499): error C2039: 'zf' n'est pas membre de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(88): note: voir la déclaration de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(499): error C3861: 'zipCloseFileInZip' : identificateur introuvable
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(500): error C2065: 'ZIP_OK' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(511): error C2039: 'zf' n'est pas membre de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(88): note: voir la déclaration de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(511): error C2065: 'zi' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(511): error C2065: 'Z_DEFLATED' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(511): error C2065: 'MAX_WBITS' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(511): error C2065: 'DEF_MEM_LEVEL' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(511): error C2065: 'Z_DEFAULT_STRATEGY' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(511): error C3861: 'zipOpenNewFileInZip3_64' : identificateur introuvable
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(512): error C2065: 'ZIP_OK' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(519): error C2039: 'zf' n'est pas membre de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(88): note: voir la déclaration de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(519): error C3861: 'zipWriteInFileInZip' : identificateur introuvable
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(520): error C2065: 'ZIP_OK' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(527): error C2039: 'zf' n'est pas membre de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(88): note: voir la déclaration de '`anonymous-namespace'::zipFileHandler'
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(527): error C3861: 'zipCloseFileInZip' : identificateur introuvable
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(528): error C2065: 'ZIP_OK' : identificateur non déclaré
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(356): warning C4996: 'fopen': This function or variable may be unsafe. Consider using fopen_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(370): warning C4996: 'fopen': This function or variable may be unsafe. Consider using fopen_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(378): warning C4996: 'fopen': This function or variable may be unsafe. Consider using fopen_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(467): warning C4996: 'localtime': This function or variable may be unsafe. Consider using localtime_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\modules\LIBXMLPlugin\daeLIBXMLPlugin.cpp(773): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
[139/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\src\1.5\CMakeFiles\colladadom150.dir\dom\domAccessor.cpp.obj /Fddom\src\1.5\CMakeFiles\colladadom150.dir\colladadom150.pdb /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\1.5\dom\domAccessor.cpp
[140/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\src\1.5\CMakeFiles\colladadom150.dir\dom\domAnimation.cpp.obj /Fddom\src\1.5\CMakeFiles\colladadom150.dir\colladadom150.pdb /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\1.5\dom\domAnimation.cpp
[141/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\src\1.5\CMakeFiles\colladadom150.dir\dom\domAny.cpp.obj /Fddom\src\1.5\CMakeFiles\colladadom150.dir\colladadom150.pdb /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\1.5\dom\domAny.cpp
[142/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\src\1.5\CMakeFiles\colladadom150.dir\dom\domBind_kinematics_model.cpp.obj /Fddom\src\1.5\CMakeFiles\colladadom150.dir\colladadom150.pdb /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\1.5\dom\domBind_kinematics_model.cpp
[143/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\src\1.5\CMakeFiles\colladadom150.dir\dom\domAnimation_clip.cpp.obj /Fddom\src\1.5\CMakeFiles\colladadom150.dir\colladadom150.pdb /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\1.5\dom\domAnimation_clip.cpp
[144/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\src\1.5\CMakeFiles\colladadom150.dir\dom\domAsset.cpp.obj /Fddom\src\1.5\CMakeFiles\colladadom150.dir\colladadom150.pdb /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\1.5\dom\domAsset.cpp
[145/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\src\1.5\CMakeFiles\colladadom150.dir\dom\domArticulated_system.cpp.obj /Fddom\src\1.5\CMakeFiles\colladadom150.dir\colladadom150.pdb /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\1.5\dom\domArticulated_system.cpp
[146/446] C:\PROGRA~2\MIB055~1\2019\PROFES~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\cl.exe   /TP -DDOM_INCLUDE_LIBXML -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\libxml2 -ID:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\include /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -DUSE_URIPARSER   -DCOLLADA_DOM_SUPPORT141 -DCOLLADA_DOM_SUPPORT150 -DCOLLADA_DOM_DAEFLOAT_IS64 -DCOLLADA_DOM_NAMESPACE -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB -DDOM_DYNAMIC -DDOM_EXPORT /showIncludes /Fodom\src\1.5\CMakeFiles\colladadom150.dir\dom\domAxis.cpp.obj /Fddom\src\1.5\CMakeFiles\colladadom150.dir\colladadom150.pdb /FS -c D:\cschulte\DevEnv64\vcpkg\buildtrees\collada-dom\src\1f440d20cd-258c30aa60.clean\dom\src\1.5\dom\domAxis.cpp
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

This issue is linked to the installation of libzip before. Indeed both libzip and minizip have an include file named zip.h, which is needed by src/modules/LIBXMLPlugin/daeLIBXMLPlugin.cpp. Problem is that in the compile line for this file the order of include is : 
-external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include -> path containing zip.h from libzip
-external:ID:\cschulte\DevEnv64\vcpkg\installed\x64-windows\include\minizip -> path containing zip.h from minizip

Renaming temporarly file zip.h from libzip allows to compile correctly collada as it will use the correct zip.h

I think the best solution would be to invert the include path order or modify <code>#include <zip.h></code> to <code>#include <minizip/zip.h></code>

--- a/ports/collada-dom/use-vcpkg-minizip.patch
+++ b/ports/collada-dom/use-vcpkg-minizip.patch
@@ -12,16 +12,33 @@ index f6c2bb6..3351ab0 100644
  if(minizip_FOUND)
    set(MINIZIP_INCLUDE_DIR ${minizip_INCLUDE_DIRS})
  else()
+diff --git a/dom/include/dae/daeZAEUncompressHandler.h b/dom/include/dae/daeZAEUncompressHandler.h
+index e9b0e9e..3d120da 100644
+--- a/dom/include/dae/daeZAEUncompressHandler.h
++++ b/dom/include/dae/daeZAEUncompressHandler.h
+@@ -9,7 +9,7 @@
+ #ifndef __DAE_ZAE_UNCOMPRESS_HANDLER_H__
+ #define __DAE_ZAE_UNCOMPRESS_HANDLER_H__
+
+-#include <unzip.h>
++#include <../minizip/unzip.h>
+ #include <libxml/xmlreader.h>
+ #include <dae/daeURI.h>
+
 diff --git a/dom/src/modules/LIBXMLPlugin/daeLIBXMLPlugin.cpp b/dom/src/modules/LIBXMLPlugin/daeLIBXMLPlugin.cpp
-index 4536275..77651f9 100644
+index 4536275..2666959 100644
 --- a/dom/src/modules/LIBXMLPlugin/daeLIBXMLPlugin.cpp
 +++ b/dom/src/modules/LIBXMLPlugin/daeLIBXMLPlugin.cpp
-@@ -32,7 +32,7 @@
+@@ -32,9 +32,9 @@
  #include <iomanip>
  using namespace std;
- 
+
 -#include <zip.h> // for saving compressed files
-+#include <minizip/zip.h> // for saving compressed files
++#include <../minizip/zip.h> // for saving compressed files
  #ifdef _WIN32
- #include <iowin32.h>
+-#include <iowin32.h>
++#include <../minizip/iowin32.h>
  #else
+ #include <unistd.h>
+ #endif
+ 

--- a/ports/collada-dom/use-vcpkg-minizip.patch
+++ b/ports/collada-dom/use-vcpkg-minizip.patch
@@ -41,4 +41,3 @@ index 4536275..2666959 100644
  #else
  #include <unistd.h>
  #endif
- 

--- a/ports/collada-dom/use-vcpkg-minizip.patch
+++ b/ports/collada-dom/use-vcpkg-minizip.patch
@@ -12,3 +12,16 @@ index f6c2bb6..3351ab0 100644
  if(minizip_FOUND)
    set(MINIZIP_INCLUDE_DIR ${minizip_INCLUDE_DIRS})
  else()
+diff --git a/dom/src/modules/LIBXMLPlugin/daeLIBXMLPlugin.cpp b/dom/src/modules/LIBXMLPlugin/daeLIBXMLPlugin.cpp
+index 4536275..77651f9 100644
+--- a/dom/src/modules/LIBXMLPlugin/daeLIBXMLPlugin.cpp
++++ b/dom/src/modules/LIBXMLPlugin/daeLIBXMLPlugin.cpp
+@@ -32,7 +32,7 @@
+ #include <iomanip>
+ using namespace std;
+ 
+-#include <zip.h> // for saving compressed files
++#include <minizip/zip.h> // for saving compressed files
+ #ifdef _WIN32
+ #include <iowin32.h>
+ #else

--- a/ports/collada-dom/vcpkg.json
+++ b/ports/collada-dom/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "collada-dom",
   "version": "2.5.0",
-  "port-version": 9,
+  "port-version": 10,
   "description": "The COLLADA Document Object Model (DOM) is an application programming interface (API) that provides a C++ object representation of a COLLADA XML instance document.",
   "homepage": "https://github.com/rdiankov/collada-dom",
   "license": null,

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1666,7 +1666,7 @@
     },
     "collada-dom": {
       "baseline": "2.5.0",
-      "port-version": 9
+      "port-version": 10
     },
     "colmap": {
       "baseline": "2023-03-12",

--- a/versions/c-/collada-dom.json
+++ b/versions/c-/collada-dom.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "1f17787430ea1503b479beb9734174f2fc57ba4b",
+      "version": "2.5.0",
+      "port-version": 10
+    },
+    {
       "git-tree": "d9c3011f64673cff7d61bac99def8048f4027f32",
       "version": "2.5.0",
       "port-version": 9

--- a/versions/c-/collada-dom.json
+++ b/versions/c-/collada-dom.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "646d6e3f068d54b52da0030741089be24f4cc80d",
+      "git-tree": "9efe9cf28c36df64035b505810bf68f445be62a1",
       "version": "2.5.0",
       "port-version": 10
     },

--- a/versions/c-/collada-dom.json
+++ b/versions/c-/collada-dom.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "1f17787430ea1503b479beb9734174f2fc57ba4b",
+      "git-tree": "646d6e3f068d54b52da0030741089be24f4cc80d",
       "version": "2.5.0",
       "port-version": 10
     },

